tests/nodeos_startup_catchup.py

```diff
@@ -189,6 +192,12 @@ def waitForNodeStarted(node):
         catchupNode.interruptAndVerifyExitStatus(60)
         catchupNode.popenProc=None
 
+        logFile = Utils.getNodeDataDir(catchupNodeNum) + "/stderr.txt"
+        f = open(logFile)
+        contents = f.read()
+        if contents.count("unlinkable_block_exception") > 3: # a few are fine
```

Comment:
Maybe you can even make it to 10 to prevent test flakeness.

Reply:
Lets see how it goes. If we see flakiness then we can adjust up.